tests/test_easy.py

```diff
@@ -150,16 +174,24 @@ def test_snmp_get_invalid_instance_with_abort_enabled(sess_args):
     'sess_args', [sess_v1_args(), sess_v2_args(), sess_v3_args()]
 )
 def test_snmp_get_invalid_object(sess_args):
-    res = snmp_get('iso', **sess_args)
-    assert res.snmp_type == 'NOSUCHOBJECT'
+    if sess_args['version'] == 1:
+        with pytest.raises(EasySNMPNoSuchNameError):
```

Comment:
No, because this is when `retry_no_such=False` and SNMPv1 has no way to cope. Whereas v2/v3 are perfectly fine.